app/views/posts/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Posts here</h1>
\ No newline at end of file
+<%= render 'shared/users', user: @user %>
+
+<% @posts.each  do  |post|%>
+<div class="boxes comments-container">
+    <a href="/users/<%= post.author_id%>/posts/<%= post.id %>"><%= post.title%></a>
+        <p><%= post.text%></p>
+        <p class="likes-comments">
+            Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter%>
+        </p>
+</div>
+<div class="boxes">
+<% Post.recent_comments(post).each  do |comment|%>
+    <p><%= comment.author.name%> - <%= comment.text %></p>
+    <%end%>
+</div>
+<br />
+<%end%>
+
+<div class="pagination-btn">
+<button>Pagination</button>
+</div>
```

Comment:
- [ ] **[OPTIONAL]** Also, please make sure that your `Post#index` view displays the correct information, currently there are two posts but none of them appear in this view. 🤓 🔧 

![image](https://github.com/joyapisi/blog_app/assets/77069011/31996380-a166-41bc-b360-b6b09855a96e)

Reply:
Okay. Thank you!I will work on that.